### DISCUSSION TITLE
In App messaging customization

### DISCUSSION THREAD

**Discussion by GolliAch** [invertase/react-native-firebase#7120] [Q&A]
Hi, 

I was wondering if there's a way to create my own message display library using React native. I saw it was possible in [iOS](https://firebase.google.com/docs/in-app-messaging/customize-messages?platform=ios) and Android, but can't find any documentation on doing it with React Native. 

Your help would be appreciated! 🤍

**Comment by ghost** [comment]
I'm also wondering the same... Couldn't find anyone commenting about it... 
Did you find a way @GolliAch ?
